src/contexts/membershipMintPage/MembershipMintPageContext.tsx

```diff
@@ -49,7 +49,7 @@ type Props = { children: ReactNode };
 
 const MembershipMintPageProvider = memo(({ children }: Props) => {
   const [totalSupply, setTotalSupply] = useState<number>(0);
-  const [remainingSupply, setRemainingSupply] = useState<number>(0);
+  const [remainingSupply, setRemainingSupply] = useState<number | null>(null);
```

Comment:
`remainingSupply` defaulting to 0 could trick the UI into briefly displaying that the mint has ended